application/analyst/services/tension_analyzer.py

```
@@ -40,13 +46,36 @@ async def analyze_tension(self, request: TensionSlingshotRequest) -> TensionDiag
+        )
 
-        # 5. 解析响应
-        diagnosis = self._parse_response(response)
+        try:
+            payload = await structured_json_generate(
+                llm=self.llm_client.provider,
+                prompt=prompt,
+                config=config,
+                schema_model=contract_model,
+            )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)